ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-ci-pipeline.yml

```diff
@@ -221,8 +221,10 @@ spec:
       params:
         - name: pipeline_image
           value: "$(params.pipeline_image)"
-        - name: operator_path
-          value: "$(tasks.bundle-path-validation.results.package_path)"
+        - name: affected_operators
+          value: "$(tasks.bundle-path-validation.results.package_name)"
+        - name: affected_catalog_operators
+          value: "$(tasks.detect-changes.results.affected_catalog_operators)"
```

Comment:
The `detect-changes` is not available in the ci-pipeline. The pipeline is only used to test bundles so instead of using the result, you can use `""` as a value or skip it completel,y as it already uses a default value inside of the task.